src/components/Events.js

```diff
@@ -0,0 +1,124 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card, Alert } from "react-bootstrap"
+import { useAuth } from "../contexts/AuthContext"
+import { Link, useHistory } from "react-router-dom"
+import Typography from "@material-ui/core/Typography";
+import Grid from '@material-ui/core/Grid';
+import { createTheme, ThemeProvider, styled, MuiThemeProvider } from '@material-ui/core/styles';
+import Paper from "@material-ui/core/Paper";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+/*const MyPaper = styled(Paper)(({ theme }) => ({
+  backgroundColor: '#FFFFFF',
+  elevation: 3,
+  padding: 8,
+  borderRadius: 4,
+  margin: theme.spacing(2)
+}));
+
+const MainGridContainer = styled(Grid)(({ theme }) => ({
+  margin: theme.spacing(4),
+})); */
+
+
+const Events = () => {
+
```

Comment:
Ideally, we would like our application to be accessed only by users of the platform. I would add a check here to ensure a user is logged in when accessing this page. If not, redirect the user to the login page.

This is to avoid future errors where a user tries to join an event but they are not logged in.